doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,723 @@
+
+units
+    The units of measure for the quantity, these will always be SI units,
+    with the exception that degrees are used in preference to radians
+
+axis
+    Indicates whether the coordinate should be regarded as an “X” or “Y” 
+    Cartesian coordinate.  
+
+bounds
+    Just a pointer to the variable defining the edges of the grid cells 
+
+Vertical coordinate variable 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some parameters also have a vertical scalar coordinate, height
+(in the example here, it is ``1.5 m`` representing screen level): 
+
+.. literalinclude:: temp12max_prob_ncdump.txt
+    :tab-width: 4
+    :lines: 55-58
+
+It's attributes are:
+
+standard_name
+    A descriptive name, either from the `CF Standard Name`_ list.
+
+units
+    The units of measure for the quantity, these will always be SI units.  
+
+positive
+    Indicates the direction in which values of the vertical coordinate increase,
+    i.e. where the vertical coordinate is pressure,
+    the ``positive`` attribute is ``down``
+
+
+Grid mapping variable
+*********************
+
+This describes the grid map projection.
+The example here is for the Lambert azimuthal equal area (LAEA) grid 
+used by the Met Office for the UK domain,
+but a Latitude-Longitude (strictly, equirectangular) projection
+is usually used for the global domain.
+The set of attributes will vary depending on map projection,
+so to get the exact meanings, it is best to look at 
+“Appendix F: Grid Mappings” in the `CF Metadata Conventions`_
+
+.. literalinclude:: temp12max_prob_ncdump.txt
+    :tab-width: 4
+    :lines: 15-23
+
+Other forms of IMPROVER data
+----------------------------
+
+Percentiles
+***********
+The example explored so far in the document describes the metadata
+for probabilities of exceeding a threshold, 
+but IMPROVER can also generate forecasts as a set of percentiles, 
+for which the metadata will differ slightly. 
+These difference are described below, but the full ncdump listing
+is shown in an appendix.
+
+Dimensions
+^^^^^^^^^^
+
+The ``threshold`` dimension is replaced by ``percentile`` dimension
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 2-6
+    :emphasize-lines: 2
+ 
+percentile
+    The dimension for the coordinate describing the percentile values. 
+
+Main percentile variable 
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the example here, the main variable is a 12 hour maximum temperature,
+but representing the Nth percentile of the temperature forecast probability distribution
+at each point. 
+
+.. literalinclude:: temp12max_perc_ncdump.txt
+    :tab-width: 4
+    :lines: 8-14
+
+The variable attributes are the same as the probability example
+previously described in detail, except: 
+
+standard_name
+    Can be used rather than a ``long_name``, as ``air_temperature``
+    exists as a descriptive name in the `CF Standard Name`_ list. 
+
+cell_methods
+    Used to describe statistical processing applied to the quantity
```

Comment:
I think this needs re-wording. Would the following suggestion work?

```suggestion
Used to describe statistical post-processing applied to the quantity. Cell methods no longer require the additional non-standardized part, ''(comment: of air_temperature)'', as they now refer to the main variable that is a maximum temperature over 12 hours.
```

Reply:
I think I've updated this as suggested.